Builder/Admin/ListBuilder.php

```diff
@@ -251,7 +251,7 @@ protected function fillExportActions()
       if (!count($export)) return;
 
       foreach ($export as $keyName => $params ) {
-          if (isset($params['show_button']) && filter_var($params['show_button'], FILTER_VALIDATE_BOOLEAN)) {
+          if (!isset($params['show_button']) && (isset($params['show_button']) && filter_var($params['show_button'], FILTER_VALIDATE_BOOLEAN))) {
```

Comment:
@ksn135 This rule always evaluates to false, as one of the statement will be the opposite of the other.

Change it to 
```php
if (!isset($params['show_button']) || (isset($params['show_button']) && filter_var($params['show_button'], FILTER_VALIDATE_BOOLEAN))) {
```
and it should work :smile: